cf_units/_udunits2_parser/README.md

```diff
@@ -0,0 +1,33 @@
+# cf-units parser of UDUNITS-2 grammar
+
+This sub-package is an attempt to re-implement the
+[UDUNITS-2 grammar](https://www.unidata.ucar.edu/software/udunits/udunits-2.0.4/udunits2lib.html#Grammar)
+in an implementation agnostic manner, using the power of ANTLRv4.
+
+The [lexer](udunits2Lexer.g4.jinja) is a Jinja2 template which expands
+the multi-mode tokenizer of the UDUNITS-2 grammar.
+The [parser](udunits2Parser.g4) defines the grammar of the tokens, along with
+a number of convenient lexical elements.
+
+
```

Comment:
@pelson Nuke the extra blank line...

Reply:
Did the opposite and added some more blank lines to other paragraphs (improves readability)